From abf9716be58a4b2a2fdcc69e2d320a4cb461c8c1 Mon Sep 17 00:00:00 2001
From: Ryan Johnson <johnsonryan@vmware.com>
Date: Tue, 5 Oct 2021 23:34:14 -0400
Subject: [PATCH 01/14] Correct brand and project names in `installer.py`

Correct brand and project names:
- "Vmware" to "VMware" in "Install packages on VMware virtual machine if requested"
- "VMWare" to "VMware" in "VMware Photon installer repo"
- "VMware Photon" to "VMware Photon OS" in "VMware Photon OS installer repo"
---
 photon_installer/installer.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/photon_installer/installer.py b/photon_installer/installer.py
index aea95aa..1b3bd71 100755
--- a/photon_installer/installer.py
+++ b/photon_installer/installer.py
@@ -850,7 +850,7 @@ class Installer(object):
 
     def _add_packages_to_install(self, package):
         """
-        Install packages on Vmware virtual machine if requested
+        Install packages on VMware virtual machine if requested
         """
         self.install_config['packages'].append(package)
 
@@ -861,7 +861,7 @@ class Installer(object):
         keepcache = False
         with open(self.tdnf_repo_path, "w") as repo_file:
             repo_file.write("[photon-local]\n")
-            repo_file.write("name=VMWare Photon installer repo\n")
+            repo_file.write("name=VMware Photon OS installer repo\n")
             if self.rpm_path.startswith("https://") or self.rpm_path.startswith("http://"):
                 repo_file.write("baseurl={}\n".format(self.rpm_path))
             else:
-- 
2.23.3

